src/node_constants.cc

```diff
@@ -1115,6 +1129,9 @@ void DefineConstants(Handle<Object> target) {
   DefineOpenSSLConstants(target);
   DefineSystemConstants(target);
   DefineUVConstants(target);
+#if HAVE_OPENSSL
+  DefineCryptoConstants(target);
+#endif
```

Comment:
Style suggestion: drop the #if guard here and move them inside the function in DefineCryptoConstants().